docs/modules/storage/pages/high-density-memory.adoc

```diff
@@ -166,106 +166,6 @@ To extend the memory available to the High-Density Memory Store, you can use per
-
-The optional `persistent-memory` element in the `native-memory` configuration block enables the use of the persistent memory device and defines the directories where this memory is mounted along with its operational mode.
-
-**Declarative Configuration:**
-
-[tabs] 
-==== 
-XML:: 
-+ 
--- 
-[source,xml]
-----
-<hazelcast>
-    ...
-    <native-memory allocator-type="POOLED" enabled="true">
-        <size unit="GIGABYTES" value="100" />
-        <persistent-memory enabled="true" mode="MOUNTED">
-            <directories>
-                <directory numa-node="0">/mnt/pmem0</directory>
-                <directory numa-node="1">/mnt/pmem1</directory>
-            </directories>
-        </persistent-memory>
-    </native-memory>
-    ...
-</hazelcast>
-----
---
-
-YAML::
-+
-[source,yaml]
-----
-hazelcast:
-  native-memory:
-    enabled: true
-    allocator-type: POOLED
-    size:
-      unit: GIGABYTES
-      value: 100
-    persistent-memory:
-      enabled: true
-      mode: MOUNTED    
-      directories:
-        - directory: /mnt/pmem0
-          numa-node: 0
-        - directory: /mnt/pmem1
-          numa-node: 1
-----
-====
-
-**Programmatic Configuration:**
-
-[source,java]
-----
-Config config = new Config();
-NativeMemoryConfig memoryConfig = new NativeMemoryConfig()
-                .setEnabled(true)
-                .setSize(new MemorySize(100, MemoryUnit.GIGABYTES))
-                .setAllocatorType(POOLED);
-
-PersistentMemoryConfig pmemConfig = memoryConfig.getPersistentMemoryConfig()
-                .setEnabled(true)
-                .setMode(MOUNTED)
-                .addDirectoryConfig(new PersistentMemoryDirectoryConfig("/mnt/pmem0", 0))
-                .addDirectoryConfig(new PersistentMemoryDirectoryConfig("/mnt/pmem1", 1));
-
-config.setNativeMemoryConfig(memoryConfig);
-----
-
-The following elements and attributes are also used in the configuration: 
-
-* `enabled`: Specifies whether use of memory on the persistent memory device is enabled or not. The default value is `false`, that is use of the device is disabled.
-* `mode`: Defines the operational mode of the persistent memory device. Two modes are supported:
-** `MOUNTED`: If you choose this mode, the memory is mounted into the file system (aka FS DAX).
-** `SYSTEM_MEMORY`: If you choose this mode, Hazelcast uses the already onlined persistent memory portion of the system memory (aka KMEM DAX).
-* `directories`: A list of the mounted directories for the persistent memory device which are available for the storage of all data structures backed by the High-Density Memory Store. When you specify the mounted directories, the following rules apply:
-** Use of memory on the persistent memory device is enabled automatically, you do not need to explicitly
-set the `enabled` attribute to `true`
-** Set the `mode` to `MOUNTED`.
-+
-If you don't specify a directory for the persistent memory device, standard RAM is used instead.
-
```

Comment:
We should remove the below parts as well, until - including - "Performance Impacts on Different Use Cases". All below related to our Persistent Memory support.

Reply:
Hi @blazember, I think I've removed all the relevant sections.